### DISCUSSION TITLE
[Feature] Extend Map-View with Grid-Gallery for selected locations

### DISCUSSION THREAD

**Discussion by techn0pia** [immich-app/immich#4864] [Feature Request]
### The feature

Hi all,

I really love the map features! Especially to showcase my travel destinations, so I click via map on a city (e.g. New York) and now I would like to show people my adventure here. However, in the moment only a Lightbox-Gallery with Next/Previous-button is shown, this makes presentation and finding specific photos less convenient. 

I really would love a Grid-Gallery, that displays all the photos of a selected location via overlay.
Here examples of PhotoPrism and Nextcloud Memories as Inspiration:

Possible mechanics: 
Option 1: Display all photos of the current map field of view in a grid gallery
Option 2: Display all photos of the selected bubble in a grid gallery

Example:
PhotoPrism:
![image](https://github.com/immich-app/immich/assets/119968680/3516f58b-1fb7-408e-8bfc-b9c7a317ad29)

Nextcloud Memories:
![image](https://github.com/immich-app/immich/assets/119968680/e34c5dd0-99b6-4f61-91bb-b20c6ceeb431)

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4519

**Comment by Fifou2202** [comment]
+1 – I strongly support this feature.

I often use the map to find photos taken at very specific locations using GPS metadata. Right now, clicking on a location only shows one photo, and I have to swipe through manually.

It would be incredibly useful to view all photos in a gallery/grid mode, especially to bulk delete pictures from precise spots (e.g., test shots, duplicates, random locations). Much more accurate than filtering by city name. Thanks!
